internal/internal_task_pollers.go

```diff
@@ -526,11 +526,11 @@ func (lath *localActivityTaskHandler) executeLocalActivityTask(task *localActivi
 		}
 	}()
 
-	timeout := task.params.ScheduleToCloseTimeoutSeconds
-	if task.params.StartToCloseTimeoutSeconds != 0 && task.params.StartToCloseTimeoutSeconds < timeout {
-		timeout = task.params.StartToCloseTimeoutSeconds
+	timeout := task.params.ScheduleToCloseTimeout
+	if task.params.StartToCloseTimeout != 0 && task.params.StartToCloseTimeout < timeout {
```

Comment:
Do we nill check these?

Reply:
These come from options and they are not nillable.